tests/test_logging.py

```diff
@@ -29,63 +29,57 @@ def test_defaults(ingest_pipeline):
 
     # Test that default log params were set on config
     log_dir = ingest_pipeline.data_ingest_config.log_dir
-    assert log_dir == default_log_dir
+    assert log_dir == TEST_LOG_DIR
     assert ingest_pipeline.data_ingest_config.log_level == DEFAULT_LOG_LEVEL
     assert (ingest_pipeline.data_ingest_config.overwrite_log ==
             DEFAULT_LOG_OVERWRITE_OPT)
 
     # Test that ingest.log was created
-    ingest_pipeline.run(target_api_config_path)
-    log_filepath = os.path.join(default_log_dir, DEFAULT_LOG_FILENAME)
+    ingest_pipeline.run()
+    log_filepath = os.path.join(TEST_LOG_DIR, DEFAULT_LOG_FILENAME)
     assert os.path.isfile(log_filepath) is True
 
 
-def test_log_dir(ingest_pipeline):
+def test_log_dir():
```

Comment:
I wrote all the logging tests were before I knew how to properly use pytest tmpdir lol
```suggestion
def test_log_dir(tmpdir):
    """
    Test non-default log dir
    """
    log_dir = tmpdir.mkdir('my_logs')
    ingest_pipeline = make_ingest_pipeline(log_dir=log_dir)

    # Run and generate logs
    ingest_pipeline.run()

    # User supplied log dir should exist
    assert len(os.listdir(log_dir)) == 1
```

Reply:
👍